### DISCUSSION TITLE
后台还是有问题

### DISCUSSION THREAD

**Discussion by Panda-995** [SinKy-Yan/zhenxunbot-docker#25] [General]
点登录后没反应，

**Comment by SinKy-Yan** [comment]
你设了真寻的地址吗

**Reply by Panda-995** [reply]
webui后台，真寻是正常运转，但是webui登陆显示未知错误

**Reply by SinKy-Yan** [reply]
不是，我说那个webui后台右上角那里面设了真寻的地址吗

**Reply by Panda-995** [reply]
没有，因为都是一个服务器上，我直接没有填写

**Reply by SinKy-Yan** [reply]
容器IP是分开的，要写

**Reply by 15219627356** [reply]
这样的话，后台是不是只能在宿主机访问，如果部署到阿里云上的话，后台就用不了了

**Reply by SinKy-Yan** [reply]
填目标url地址就行了啊，你阿里云服务器不也有地址

**Reply by 15219627356** [reply]
直接填阿里云服务器ip+8080端口就行了咩，我试过了不行，唔，如果是容器ip+8080端口，容器ip外网肯定访问不了的

**Reply by SinKy-Yan** [reply]
iptables啥的设一个转发，你容器如果network不是host的话外网访问进去应该是不转发的

**Reply by 15219627356** [reply]
OK谢谢啦
